### PR TITLE
docs(core): adding tabable to the cards with list

### PR DESCRIPTION
**Pull request by Lokanathan-k**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/5776

#### Please provide a brief summary of this pull request.

adding tab index to handle the tab key accessibility from card body to card item

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [NA] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [NA] update `README.md`
- [NA] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by platon-rov** [comment]
For some reason when you're getting on the card list first item arrow down navigation works only after 2nd click - could you please fix it if it's related to the card or create an appropriate ticket if it's the list-item issue?

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
I think it would be better to deal with this in the library's code rather than in the example documentation so the developer doesn't have to type this every time.  Is there a reason it has to happen here in the examples?

**Comment by Lokanathan-k** [comment]
> I think it would be better to deal with this in the library's code rather than in the example documentation so the developer doesn't have to type this every time. Is there a reason it has to happen here in the examples?

the reason is, the card component body container can have any type of component inside, it could be list, or avatar or any other component, so it will be easier if this can be handled by user.

File: apps/docs/src/app/core/component-docs/card/examples/card-compact-example.component.html [hunk 1]
```diff
@@ -5,8 +5,8 @@ <h3 fd-card-subtitle>Card Subtitle</h3>
         <span fd-card-counter>3 of 10</span>
     </fd-card-header>
     <fd-card-content>
-        <ul fd-list [noBorder]="true" [compact]="true">
-            <li fd-list-item>
+        <ul fd-list [noBorder]="true" [compact]="true" tabindex="-1">
+            <li fd-list-item tabindex="0">
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/card/examples/card-compact-example.component.html:9]: I don't think you are supposed to do this. Check List implementation.

**Comment by InnaAtanasova** [comment]
> > I think it would be better to deal with this in the library's code rather than in the example documentation so the developer doesn't have to type this every time. Is there a reason it has to happen here in the examples?
> 
> the reason is, the card component body container can have any type of component inside, it could be list, or avatar or any other component, so it will be easier if this can be handled by user.

@mikerodonnell89 has a point here. This is a problem coming from List component. You can observe the behaviour there, too.  The fix should be done in the component (List) not in the documentation of Card component.

**Comment by InnaAtanasova** [comment]
Please open this PR again when you address the comments. Thanks

**Pull request closed without merging**